### ISSUE TITLE
MudSelectExtended: Virtualize issue with ItemContent

### ISSUE THREAD

**Issue by jbartlettii** [CodeBeamOrg/CodeBeam.MudExtensions#486]
#109 #322 Are both similar but look like they are more focused on the issues with rendering with multiselect content, there is an issue with ItemContent where the value is definitely bound, but not presented on initialization.

You can see below where the value is selected in the dropdown but is not filled out in the parent Select, selecting a different item will cause the presenter to work normally, but selecting the same value does nothing. Removing the virtualize option and setting the flag to false solves the issue completely.

![image](https://github.com/user-attachments/assets/a07c9ff1-f68b-4c99-bb3e-47a3dab7e5c6)

```
<MudSelectExtended T="Asset" ItemCollection="_assets" ValuePresenter="ValuePresenter.ItemContent"
                   SearchBox="true" SearchFunc="SearchFunc" Required="true"
                   Placeholder="Assets" Label="Select Asset" Variant="Variant.Outlined"
                   Value="@Asset" ValueChanged="Callback"
                   Virtualize="true"
                   AnchorOrigin="Origin.BottomCenter">
```

Code above is what is being used, I'll try some of the workarounds used for the multiselection items to see if those help.

**Comment by panoskentros** [comment]
I still haven't found a solution in big 2026

**Comment by mckaragoz** [comment]
Could you provide a simple code for debug? It's an edge-case we need some more information about this bug before going deep.

**Comment by mckaragoz** [comment]
Could you try v 9.0.0-preview.3? It should fixed with the release

**Comment by panoskentros** [comment]
> Could you try v 9.0.0-preview.3? It should fixed with the release

Yeah it fixed it! We were also talking on discord. Thanks a lot !!

**Comment by mckaragoz** [comment]
Please re-open this issue or new issue if you have further problems.
